src/steps/mdx-to-gridtables.ts

```diff
@@ -141,15 +125,16 @@ export default function mdxToBlocks(ctx: Helix.UniversalContext) {
       });
     }
 
-    const rowsToInsert = listToMatrix(slotsToInsert, slots.length);
+    // Calculate rows to insert based on total slots
+    const rowsToInsert = totalSlots > 0 ? listToMatrix(slotsToInsert, slots.length) : [];
```

Comment:
i think we should just. use isHorizontalLine to check because there are some other components that may not have slots, right?  Are we sure we want to handle it the same way as horizontal line?